PyInstaller/hooks/hook-pubsub.core.py

```diff
@@ -0,0 +1,15 @@
+from PyInstaller.utils.hooks import collect_submodules, collect_data_files
```

Comment:
Please add a comment stating for which package it it and which version you tested. This is esp. important here to avoid confusion with wx.pubsub.